utils/tar.js

```diff
@@ -93,6 +93,25 @@ async function getContents (manifest, target) {
     })
   ])
 
+  const comparator = (a, b) => {
+    return a.path.localeCompare(b.path, undefined, {
+      sensitivity: 'case',
+      numeric: true
+    })
+  }
+
+  const isUpper = (str) => {
+    const ch = str.charAt(0)
+    // eslint-disable-next-line
+    return 'A' <= ch && ch <= 'Z'
```

Comment:
It is quite sad how standard makes writing such common idioms in a broken way, compare this:

```diff
-    return ch >= 'A' && ch <= 'Z'
+    return 'A' <= ch && ch <= 'Z'
```